vignettes/00-introduction.Rmd

```diff
@@ -1,3 +1,16 @@
 # (PART\*) Section 1: Vignettes {-}
 
-# Vignettes Introduction
\ No newline at end of file
+# Vignettes Introduction
+
+
+These vignettes provide a **quick start** tutorial - they quickly show you how to access and perform ‘hello world’ type introductions to different types of data in the TERRA Reference datasets. These vignettes are designed so that they don’t require any user accounts or passwords to access to the data so that they can be run as easily as possible.
+
+The vignettes currently include:
+* trait data
+* weather data
+* image data
+* how to use combine trait, weather, and image data in a single analysis
+
+In order to make these vignettes as easy to use as possible, we only use sample of the full dataset is available without an account; specifically we have made data from thirty two plots in each of two seasons available. Although we want to make these data broadly available, we want to keep track of who is using them and collect emails so that we can contact you.
+
+In order to use the vignettes, you will need to have R installed along with the extra packages that are used. Except where otherwise indicated, the packages are available on CRAN and can be installed using the syntax `install.packages('packagename')`. 
```

Comment:
They'll also need Python for the third and fourth vignettes?

Reply:
```suggestion
In order to use the vignettes, you will need to have R installed along with the extra packages that are used. 
All of the packages except 'trait' are available on CRAN and can be installed using the syntax `install.packages('packagename')`:

```r
install.packages('tidyverse', 'jsonlite', 'knitr', 'lubridate', 'raster', 'sf')
```

Although 'traits' is also on CRAN, you need to download the TERRA REF version from GitHub to handle larger datasets: 

```r
devtools::install_github('terraref/traits')
```
```